### PR TITLE
Steam Test Server - Update 9

### PR DESCRIPTION
**Pull request by Sunaries**
https://github.com/Treeofsavior/EnglishTranslation/commit/53b5c6c6b891dcc4bb70b6f9f7e67c055a2a64a1 proofread + other fixes

@imcgames 

Please check Discord :smile: . Regarding a few matters, including this:

GitHub Logo

### CODE REVIEW

**Comment by Sunaries** [comment]
Good to merge.

File: ETC.tsv [hunk 1]
```diff
@@ -17665,41 +17665,41 @@ ETC_20151224_017666	In substance, this is the kind of Golem and it is a creature
...
-ETC_20151224_017687	It is hard to distinguish them from other monsters that are transformed from other mushroom types, but they are definitely the ones that are transformed from the creatures of the devildom.
-ETC_20151224_017688	In some regions, they created either Kupols or the idols of the spirits. Some of them started to work as evil creatures after Medizo Diena.
+ETC_20151224_017675	A monster that the sorcerers of the devil realm called out using noir. There's a rumor that they use the spirits in the forest after capturing them, but that has yet to be confirmed.
+ETC_20151224_017676	It is not uncommon that the stone statues that are engraved with magical power or parts of architectures move to harm humans. After Medzio Diena, there's no one who think that could be a bit strange.
+ETC_20151224_017677	Cryomancer Master Aleister Crowley
+ETC_20151224_017678	Until the year after Medzio Diena, people thought that the monsters that turned from the crops that haven't been harvested would not appear again if harvestors would be careful. However, that expectation was gone after the monsters grew by themselves. Pumpleflap is an example of this.
```
Comment (grillo-delmal) [inline ETC.tsv:17676]: ... would not appear again if harvestors would be careful.
... would not appear again if **harvesters were careful**.

and

However, that expectation was gone after the monsters grew by themselves.
However, **they lost their hope when the monsters started growing** by themselves.

File: ETC.tsv [hunk 2]
```diff
@@ -17665,41 +17665,41 @@ ETC_20151224_017666	In substance, this is the kind of Golem and it is a creature
...
-ETC_20151224_017696	Elma is one of the Gesti's bodyguards and she is dispatched to various duties and she also works as the guard at the devildom.
+ETC_20151224_017693	A long time ago, Elet used to be a fairy or a soul. They received the curse of Goddess Ausrine and they hid into the human world and lived on. Since Medzio Diena, when the demons invaded the world, they are showing their determination to participate with the other demons.
+ETC_20151224_017694	What's unique about Nuo is that they are subordinates of the Demon King Kartas. Although Kartas possesses a brutal and unexpected personality, it is so incredible that Nuo who is under the control of Kartas possesses the power and the number for the ones who know well about the circumstances in the demons.
+ETC_20151224_017695	Nuka is under the control of the Demon King Vaiga and was dispatched here to fulfill the tasks that Gesti received from Giltine. But, Gesti is controlling them unlike what Vaiga once thought.
+ETC_20151224_017696	Elma is one of the Gesti's bodyguards and she is dispatched to various duties and she also works as the guard at the devil realm.
```
Comment (grillo-delmal) [inline ETC.tsv:17694]: Elma is one of the Gesti's bodyguards and she is dispatched to various duties and she also works as the guard at the devil realm.

Elma is one **of Gesti's bodyguards, she** is dispatched to various duties **and also works** as **a guard at** the devil realm.

File: ETC.tsv [hunk 3]
```diff
@@ -17665,41 +17665,41 @@ ETC_20151224_017666	In substance, this is the kind of Golem and it is a creature
...
+ETC_20151224_017698	Bloks just appeared within the last year, so there isn't much information about it yet.
+ETC_20151224_017699	There are quite a few reckless people who are going after the monsters that have big ores like the Kenol attached onto the bodies of the monsters. It would have been better if they had acted more cautiously when thinking about the disaster that was brought to the world.
+ETC_20151224_017700	He has four strong arms. If you don't be careful, you may receive major damage.
+ETC_20151224_017701	A demon that follows the Demon Lord Hauberk. Most of them still show their loyalty to Hauberk even after their owner was sealed by Helgasercle.
+ETC_20151224_017702	Hohen competed against the humans since the birth of the kingdom. They were cooperative to demons instead of the goddesses, but they posed no big problems since their amount was small compared to humans. However, the environment that Medzio Diena changed is working more advantegeous for Hohens instead of humans so it is not certain whether humans would have an edge over them in the future.
```
Comment (grillo-delmal) [inline ETC.tsv:17700]: Hohen competed against the humans since....

Hohen competed **against humans** since...

File: QUEST.tsv [hunk 4]
```diff
@@ -3147,7 +3147,7 @@ QUEST_20151224_003154	(The pillar shows an image of a goddess with the number 3.
 QUEST_20151224_003155	(The pillar shows an image of a human with the number 3.)
 QUEST_20151224_003156	(The pillar shows an image of a Demon Goddess with the number 3.)
 QUEST_20151224_003157	...Do you see it?{nl}
-QUEST_20151224_003158	This is the catastrophe that already fell upon us.{nl}But an ever greater calamity is just beginning.{nl}
+QUEST_20151224_003158	This is the catastrophe that already fell upon us...{nl}But an ever greater calamity is just the beginning.{nl}
```
Comment (grillo-delmal) [inline QUEST.tsv]: This text is the same as on line QUEST_20151224_003303, but the korean texts are different.

File: QUEST.tsv [hunk 5]
```diff
@@ -3292,11 +3292,11 @@ QUEST_20151224_003299
 QUEST_20151224_003300	
 QUEST_20151224_003301	
 QUEST_20151224_003302	
-QUEST_20151224_003303	This is the catastrophe that already fell upon us.{nl}But an ever greater calamity is just beginning.{nl}
-QUEST_20151224_003304	But... Where could the goddess of destiny be?{nl}And the other goddesses...?{nl}
-QUEST_20151224_003305	See?{nl}No goddesses can protect you from death.{nl}
-QUEST_20151224_003306	That's why the Revelator...{nl}I close my eyes and ears...
-QUEST_20151224_003307	Just run away like everyone else did.
+QUEST_20151224_003303	This is the catastrophe that already fell upon us...{nl}But an ever greater calamity is just the beginning.{nl}
```
Comment (grillo-delmal) [inline QUEST.tsv]: But an ever greater calamity is just the beginning.

But an **even** greater calamity is **just beginning**.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
Please check Discord :+1:

**Pull request merged**